runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -56,22 +59,12 @@
     /**
      * A cache for read results.
      */
-    final LoadingCache<Long, ILogData> readCache = CacheBuilder.newBuilder()
+    final Cache<Long, ILogData> readCache = CacheBuilder.newBuilder()
             .maximumSize(runtime.getParameters().getNumCacheEntries())
             .expireAfterAccess(runtime.getParameters().getCacheExpiryTime(), TimeUnit.SECONDS)
             .expireAfterWrite(runtime.getParameters().getCacheExpiryTime(), TimeUnit.SECONDS)
             .recordStats()
-            .build(new CacheLoader<Long, ILogData>() {
-                @Override
-                public ILogData load(Long value) throws Exception {
-                    return cacheFetch(value);
```

Comment:
You can remove the implementation of this method as it's not used.

Reply:
cacheFetch actually had extra logic, which checks if the ILogData type is empty. I collapsed the two implementations, this adds more consistency.